pkg/reconciler/parallel/controller.go

```diff
@@ -42,15 +46,25 @@ func NewController(
 
 	parallelInformer := parallel.Get(ctx)
 	subscriptionInformer := subscription.Get(ctx)
+	eventPolicyInformer := eventpolicy.Get(ctx)
 
 	r := &Reconciler{
 		parallelLister:     parallelInformer.Lister(),
 		subscriptionLister: subscriptionInformer.Lister(),
 		dynamicClientSet:   dynamicclient.Get(ctx),
 		eventingClientSet:  eventingclient.Get(ctx),
+		eventPolicyLister:  eventPolicyInformer.Lister(),
 	}
 	impl := parallelreconciler.NewImpl(ctx, r)
 
+	var globalResync func()
+	featureStore := feature.NewStore(logging.FromContext(ctx).Named("feature-config-store"), func(name string, value interface{}) {
+		if globalResync != nil {
+			globalResync()
+		}
+	})
+	featureStore.WatchConfigs(cmw)
```

Comment:
Nice! We just need to use this store in the `parallelreconciler.NewImpl(ctx, r, ...options)` now (line 58). See for example how that is done here: https://github.com/knative/eventing/blob/bb2e0a30ff68c7f145ef243451c446e1a471b3dc/pkg/reconciler/inmemorychannel/controller/controller.go#L105-L109

Reply:
sure, thanks for reminding @Cali0707 👍🏼

Reply:
done, PTAL!